WordPress/src/main/res/color/primary_emphasis_medium_selector.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:alpha="@dimen/material_emphasis_disabled" android:color="@color/colorPrimary" android:state_pressed="true" />
+    <item android:color="@color/colorPrimary" />
+</selector>
+
```

Comment:
Minor (🔍): Consider removing the extra line at the end of this file.

Reply:
In 4d86bec